src/api/main.py

```diff
@@ -105,5 +172,30 @@ def load_json(shp_path, output_path, simplify_tolerance=None):
         log.exception('Could not get GeoJSON from output.')
         return None
 
-if __name__ == "__main__":
-    app.run(host="0.0.0.0")
+
+def reproject_point(lat_lon, from_epsg, to_epsg):
+    """
+    Source: http://gis.stackexchange.com/a/78850
+    """
+    lat, lon = lat_lon
+
+    lat = float(lat)
+    lon = float(lon)
+
+    point = ogr.Geometry(ogr.wkbPoint)
+    point.AddPoint(lon, lat)
+
+    inSpatialRef = osr.SpatialReference()
+    inSpatialRef.ImportFromEPSG(from_epsg)
+
+    outSpatialRef = osr.SpatialReference()
+    outSpatialRef.ImportFromEPSG(to_epsg)
+
+    coordTransform = osr.CoordinateTransformation(inSpatialRef, outSpatialRef)
+    point.Transform(coordTransform)
+
+    return point.GetY(), point.GetX()
+
+
+if __name__ == '__main__':
+    app.run(host='0.0.0.0', debug=True)
```

Comment:
We use gunicorn, not the flask debug server in production. The `debug` flag should only be used during development.

Ref: https://github.com/WikiWatershed/docker-rwd/blob/develop/Dockerfile#L68